src/Vezeeta.Infrastructure/Data/AppDbContext.cs

```diff
@@ -1,16 +1,23 @@
-﻿using Microsoft.EntityFrameworkCore;
+﻿
+using Microsoft.AspNetCore.Identity.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore;
 using System.Reflection;
 using Vezeeta.Domain.Entities;
 
 namespace Vezeeta.Infrastructure.Data;
 
-public sealed class AppDbContext : DbContext
+public sealed class AppDbContext : IdentityDbContext<ApplicationUser>
 {
     public AppDbContext(DbContextOptions options) : base(options)
     {
     }
 
-    public DbSet<Doctor> Doctors { get; set; }
+    DbSet<Doctor> Doctors { get; set; }
+    DbSet<Specialization> Specializations { get; set; }
+    DbSet<Times> Times { get; set; }
+    DbSet<Appointment> Appointments { get; set; }
+    DbSet<Request> Requests { get; set; }
+    DbSet<DiscountCode> DiscountCodes { get; set; }
```

Comment:
The `DbSet<T>` should be `public` in order to access them, currently you cannot as he default is `private`

#OOP_Encapsulation